cicd-pipelines/manual-approve-with-github-actions/github-actions.yml

```diff
@@ -39,14 +40,14 @@ jobs:
       - name: Install Terraform
         uses: hashicorp/setup-terraform@v2.0.0
         with:
-            terraform_version: '1.0.11'
-      
+          terraform_version: $TERRAFORM_VERSION
+
       - name: Set up Databricks CLI profile for Terraform provider
         run: |
-            echo "[DEFAULT]" >> ~/.databrickscfg
-            echo "host = $DATABRICKS_HOST" >> ~/.databrickscfg
-            echo "token = $DATABRICKS_TOKEN" >> ~/.databrickscfg
-                    
+          echo "[DEFAULT]" >> ~/.databrickscfg
```

Comment:
can you test with removing this CLI profile? it should pick up `DATABRICKS_HOST`/`DATABRICKS_TOKEN` environment variables automatically

Reply:
done in https://github.com/databricks/terraform-databricks-examples/pull/71/commits/351823bbe3d6a027539f4801feaa952758e4d5a6